src/popup/pages/TransactionDetails.vue

```diff
@@ -227,17 +227,18 @@ export default defineComponent({
     const getTxTipUrl = useGetter('getTxTipUrl');
     const getExplorerPath = useGetter('getExplorerPath');
     const isTxAex9 = useGetter('isTxAex9');
+    const getTxSymbol = useGetter('getTxSymbol');
+    const getTxAmountTotal = useGetter('getTxAmountTotal');
 
     const tipUrl = computed(() => getTxTipUrl.value(transaction.value));
+
     const {
       setTransaction,
-      getTxSymbol,
-      getTxAmountTotal,
       isErrorTransaction,
       isAllowance,
       tokens,
       isDex,
-    } = useTransactionToken(undefined, true);
+    } = useTransaction({ showDetailedAllowanceInfo: true });
```

Comment:
Can you please move the composable initiation to the top of the setup?

Reply:
At the exact top or maybe should it stay after getters?